### PR TITLE
Use webpack to bundle the marker static resources.

### PR DESCRIPTION
**Pull request by SylvainCorlay**
@blink1073 this is what we talked about. I added some comments about the webpack issues in the webpack config file.

This is the version that generates the corrupted files.

### CODE REVIEW

File: js/webpack.config.js [hunk 1]
```diff
@@ -3,7 +3,24 @@ var version = require('./package.json').version;
 var loaders = [
     { test: /\.json$/, loader: "json-loader" },
     { test: /\.css$/, loader: "style-loader!css-loader" },
+    // Generic file loader, Should be used for anything but leaflet's
+    // marker-icon.png, marker-icon-2x.png or marker-shadow.png
     { test: /\.(jpg|png|gif)$/, loader: "file" },
+    // Files marker-icon.png, marker-icon-2x.png or marker-shadow.png
```
Comment (blink1073) [inline js/webpack.config.js]: You can add an `exclude` option to the previous loader, no?

File: js/webpack.config.js [hunk 2]
```diff
@@ -3,7 +3,24 @@ var version = require('./package.json').version;
...
+    // var path = L.Icon.Default.imagePath;
+    //
+    //  if (!path) {
+    //      throw new Error('Couldn\'t autodetect
+    //      L.Icon.Default.imagePath, set it manually.');
+    //  }
+    //
+    //  return path + '/marker-' + name + '.png';
+    //
+    { test: /\.(png)$/, loader: "file?name=[name].[ext]" }
```
Comment (blink1073) [inline js/webpack.config.js]: And then add `marker` to the regex here.

File: js/webpack.config.js (hunk 1 above)
Comment (blink1073) [inline js/webpack.config.js]: http://webpack.github.io/docs/configuration.html#module-loaders

**Comment by SylvainCorlay** [comment]
Thanks to @blink1073 I could get the marker-static assets to work!

**Comment by blink1073** [comment]
:fist: :tada:

**Pull request merged**